testing/mysql/integration_test.go

```diff
@@ -33,6 +33,9 @@ import (
 	"cloud.google.com/go/spanner"
 	database "cloud.google.com/go/spanner/admin/database/apiv1"
 	"google.golang.org/api/iterator"
+	storage "cloud.google.com/go/storage"
+	"google.golang.org/api/option"
+	fakestorage "github.com/fsouza/fake-gcs-server/fakestorage"
```

Comment:
I will need to check on the feasibility of using this package in code here, for license and security concerns.

Reply:
Understandable, if it's not feasible should I create my own fake storage for Cloud Storage? As Cloud Storage doesn't have an emulator, so I used fakestorage to create a fake GCS server on port 9000